### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_AzureDataStudio-Insiders_ColorContrast: The focus on 4 links is below 3:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#10616]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#Benchmark;#DesktopApp;#MAS1.4.11;#SQL Azure Data Studio;#FTP;#ColorContrast;#Win32;#Acc123;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.4.11
Tool: Accessibility insights for desktop

**Repro Steps:**
1 .Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 .Launch Accessibility insights for desktop and select third tab contrast.

**Actual:**
When the tab focus in on " create a connection, run a query, create a notebook, deploy a server " the focus color contrast is at 2.227:1 which is below 3:1

**Same issue for the following page:**
1 .Resources- tab focus on show more and get started and documentation link.
2. Extend your data Studio-all controls in the section beside install button
3 .Videos Pop Up- on both links 

**Expected:**
The color contrast of the focus should be 3:1 or greater 

**User Impact:**
Low vision users will face difficulty in viewing the focus.

**Attachment link for Reference:**
![10616A11y_AzureDataStudioStayHealthy_Welcome_AzureDataStudio-Insiders_ColorContrast](https://user-images.githubusercontent.com/24580570/83235836-25e06380-a1b0-11ea-99e1-d6aff04a8dd7.png)

**Comment by kisantia** [comment]
@fsteffi the contrast ratio is above 3:1 when the focus is on these links in the latest insiders build:
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/9ba2cfee-5a25-4d4c-924e-7f4ac3a94bec)

Please verify and close this issue.
